arm_joint_topic: /uarm/move_to_joints
arm_status_topic: /uarm/joint_state
arm_cartesian_up_topic: /ras_group8_arm/cartesian_up
arm_cartesian_down_topic: /ras_group8_arm/cartesian_down

arm_pump_topic: /uarm/pump

arm_dim:
  a: 10.0
  b: 2.0
  c: 15.0
  d: 16.0
  e: 3.5
  f: -6

arm_angles: 
 j0_zero: 88 
 j1_zero: 23
 j2_v: 96

arm_base_position:
 x: 0
 y: 15 
 z: 5
 

arm_movement: 
 move_time: 2


